l10n_it_fiscal_document_type/tests/test_doc_type.py

```diff
@@ -77,8 +113,35 @@ def test_doc_type_on_invoice_create(self):
                 ],
             }
         )
+        self.assertEqual(invoice.move_type, "out_refund")
         self.assertEqual(invoice.fiscal_document_type_id, self.TD04)
 
+        invoice_form = Form(
+            self.inv_model.with_context(default_move_type="out_invoice")
+        )
+        invoice_form.partner_id = self.partner4
+        with invoice_form.invoice_line_ids.new() as line:
+            line.product_id = product
+            line.quantity = 10.0
+            line.account_id = revenue_account
+            line.name = "product test 5"
+            line.price_unit = 100.00
+        invoice = invoice_form.save()
```

Comment:
```suggestion
        invoice = self.init_invoice("out_invoice", partner=self.partner4)
```
Potresti sintetizzare un po' la creazione di queste nuove fatture?
Già che le stai modificando magari si può fare anche per le altre fatture del test, che ne pensi?

Questo test ora è di 100 righe e secondo me si inizia a perdere un po' il filo della sua utilità: potresti indicare quali casi ci sono già e quali stai aggiungendo? Perché così si rischia di rifare gli stessi test diverse volte.
Altrimenti, forse meglio aggiungere i nuovi casi in dei test dedicati.

Reply:
Fatto, grazie